packages/core/data-transfer/lib/providers/local-strapi-destination-provider/__tests__/restore.test.ts

```diff
@@ -113,7 +114,7 @@ describe('Restore ', () => {
         query,
       },
     })();
-    const config = {
+    const config: IConfiguration = {
```

Comment:
I thought this would raise a compilation error on value being unknown but assign an object value :thinking: 
I think I wasn't allowed to do just that in the entities PR :thinking:

Reply:
Weird, in this case I actually got an error because it didn't have it.